### PR TITLE
Write boolean value when saving a form (bug 1729971)

### PR DESCRIPTION
**Pull request by calixteman**
- it aims to fix https://bugzilla.mozilla.org/show_bug.cgi?id=1729971#c4.

### CODE REVIEW

File: src/core/writer.js [hunk 1]
```diff
@@ -83,10 +83,14 @@ function writeValue(value, buffer, transform) {
     buffer.push(`(${escapeString(value)})`);
   } else if (typeof value === "number") {
     buffer.push(numberToString(value));
+  } else if (typeof value === "boolean") {
+    buffer.push(`${value.toString()}`);
```
Comment (Snuffleupagus) [inline src/core/writer.js:87]: I might be missing something obvious here, but why do you need to wrap this in a string?
Isn't `value.toString()` *already* a string here, in which case just `buffer.push(value.toString());` ought to work.

**Review by Snuffleupagus — APPROVED** [review]
r=me with that comment addressed; thank you!

**Comment by timvandermeij** [comment]
Looks good to me too with the comment above addressed.

File: src/core/writer.js (hunk 1 above)
Comment (Snuffleupagus) [reply src/core/writer.js:87]: Also, as a follow-up, we should probably fix the pre-existing cases above to reduce unnecessary confusion when reading this code.

**Comment by Snuffleupagus** [comment]
Actually, let's land this as-is such that it won't *just* miss the upcoming release and we can fix the comment in a follow-up.

**Pull request merged**